### ISSUE TITLE
Documentation for Yarn

### ISSUE THREAD

**Issue by WillSquire** [ranisalt/node-argon2#210]
Bites me everytime I come to use this library, Yarn just doesn't build the binaries. For those looking for a quick fix, use `npm rebuild --update-binary`. See here for more info on why: https://github.com/yarnpkg/yarn/issues/2069

EDIT:
This worked better: `npm_config_build_from_source=true yarn install --force`. The above way results in the binary being removed the next time Yarn runs.

**Comment by ranisalt** [comment]
Hm, that's peculiar from yarn. Anyway we're experimenting with prebuilt packages so that should be fixed transparently soon!

**Comment by sethgoodluck** [comment]
@WillSquire @rainisalt -- I couldn't get your fix to work for 0.25.1

Hit the following: 
```
npm rebuild --update-binary

> core-js@3.3.5 postinstall /Users/sethmartin/Projects/politicore/node_modules/core-js
> node postinstall || echo "ignore"


> protobufjs@6.8.8 postinstall /Users/sethmartin/Projects/politicore/node_modules/protobufjs
> node scripts/postinstall


> argon2@0.25.1 install /Users/sethmartin/Projects/politicore/node_modules/argon2
> node-pre-gyp install --fallback-to-build

node-pre-gyp WARN Using needle for node-pre-gyp https download 
node-pre-gyp WARN Tried to download(404): https://github.com/ranisalt/node-argon2/releases/download/v0.25.1/argon2-v0.25.1-node-v64-darwin-x64-un
known.tar.gz 
node-pre-gyp WARN Pre-built binaries not found for argon2@0.25.1 and node@10.16.3 (node-v64 ABI, unknown) (falling back to source compile with no
de-gyp) 
No receipt for 'com.apple.pkg.CLTools_Executables' found at '/'.

No receipt for 'com.apple.pkg.DeveloperToolsCLILeo' found at '/'.

No receipt for 'com.apple.pkg.DeveloperToolsCLI' found at '/'.

gyp: No Xcode or CLT version detected!
gyp ERR! configure error 
gyp ERR! stack Error: `gyp` failed with exit code: 1
gyp ERR! stack     at ChildProcess.onCpExit (/Users/sethmartin/.nvm/versions/node/v10.16.3/lib/node_modules/npm/node_modules/node-gyp/lib/configu
re.js:351:16)
gyp ERR! stack     at ChildProcess.emit (events.js:198:13)
gyp ERR! stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:248:12)
gyp ERR! System Darwin 19.0.0
gyp ERR! command "/Users/sethmartin/.nvm/versions/node/v10.16.3/bin/node" "/Users/sethmartin/.nvm/versions/node/v10.16.3/lib/node_modules/npm/nod
e_modules/node-gyp/bin/node-gyp.js" "configure" "--fallback-to-build" "--module=/Users/sethmartin/Projects/politicore/node_modules/argon2/lib/bin
ding/argon2.node" "--module_name=argon2" "--module_path=/Users/sethmartin/Projects/politicore/node_modules/argon2/lib/binding" "--napi_version=4"
 "--node_abi_napi=napi" "--napi_build_version=0" "--node_napi_label=node-v64"
gyp ERR! cwd /Users/sethmartin/Projects/politicore/node_modules/argon2
gyp ERR! node -v v10.16.3
gyp ERR! node-gyp -v v5.0.5
gyp ERR! not ok 
node-pre-gyp ERR! build error 
node-pre-gyp ERR! stack Error: Failed to execute '/Users/sethmartin/.nvm/versions/node/v10.16.3/bin/node /Users/sethmartin/.nvm/versions/node/v10
.16.3/lib/node_modules/npm/node_modules/node-gyp/bin/node-gyp.js configure --fallback-to-build --module=/Users/sethmartin/Projects/politicore/nod
e_modules/argon2/lib/binding/argon2.node --module_name=argon2 --module_path=/Users/sethmartin/Projects/politicore/node_modules/argon2/lib/binding
 --napi_version=4 --node_abi_napi=napi --napi_build_version=0 --node_napi_label=node-v64' (1)
node-pre-gyp ERR! stack     at ChildProcess.<anonymous> (/Users/sethmartin/Projects/politicore/node_modules/node-pre-gyp/lib/util/compile.js:83:2
9)
node-pre-gyp ERR! stack     at ChildProcess.emit (events.js:198:13)
node-pre-gyp ERR! stack     at maybeClose (internal/child_process.js:982:16)
node-pre-gyp ERR! stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:259:5)
node-pre-gyp ERR! System Darwin 19.0.0
node-pre-gyp ERR! command "/Users/sethmartin/.nvm/versions/node/v10.16.3/bin/node" "/Users/sethmartin/Projects/politicore/node_modules/argon2/nod
e_modules/.bin/node-pre-gyp" "install" "--fallback-to-build"
node-pre-gyp ERR! cwd /Users/sethmartin/Projects/politicore/node_modules/argon2
node-pre-gyp ERR! node -v v10.16.3
node-pre-gyp ERR! node-pre-gyp -v v0.13.0
node-pre-gyp ERR! not ok 
Failed to execute '/Users/sethmartin/.nvm/versions/node/v10.16.3/bin/node /Users/sethmartin/.nvm/versions/node/v10.16.3/lib/node_modules/npm/node
_modules/node-gyp/bin/node-gyp.js configure --fallback-to-build --module=/Users/sethmartin/Projects/politicore/node_modules/argon2/lib/binding/ar
gon2.node --module_name=argon2 --module_path=/Users/sethmartin/Projects/politicore/node_modules/argon2/lib/binding --napi_version=4 --node_abi_na
pi=napi --napi_build_version=0 --node_napi_label=node-v64' (1)
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! argon2@0.25.1 install: `node-pre-gyp install --fallback-to-build`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the argon2@0.25.1 install script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/sethmartin/.npm/_logs/2019-11-07T02_09_16_536Z-debug.log
```

**Comment by ranisalt** [comment]
It should all be fixed now.
